test/onnx/exporter/test_api.py

```diff
@@ -152,7 +157,7 @@ def test_dynamic_shapes_with_fully_specified_axes(self):
             strict=True,
         )
 
-        self.assert_export(exported_program)
+        self.assert_export(ep, strategy=None)
 
     def test_dynamic_shapes_supports_input_names(self):
         self.assert_export(
```

Comment:
@titaiwangms this test is having some issues actually. It's not using the default strategy.

Reply:
Removed